Rancher_Embedded_Retail_v0.6.adoc

```diff
@@ -69,4 +82,4 @@ Utilizing a standardized O/S plus Kubernetes image across all Point-of-Service u
 
 ** SUSE Rancher can guarantee that security software like Open Policy Agent Gatekeeper are deployed on every K3s cluster, on every Point-of-Service unit, in every retail location. Using cluster labels, a definition unique to SUSE Rancher, it can also be assured that specific security policies are enforced on every Point-of-Service unit, based on its current function. 
 
-** If a unit is moved for example, from a Point-of-Purchase function, to a receiving-and-inventory function; SUSE Rancher automatically removes all of the previous applications, security software, and policies and applies new resources to fit unit's new tasks and security posture. Performing the same switch based on antiquated bare-metal or virtualization deployment technologies would require engaging remote IT staff to perform complex and error prone redeployments of operating systems and applications.
\ No newline at end of file
+** If a unit is moved for example, from a Point-of-Purchase function, to a receiving-and-inventory function; SUSE Rancher can be used to clean up the old system and apply new resources to fit unit's new tasks and security posture. Performing the same switch based on antiquated bare-metal or virtualization deployment technologies would require engaging remote IT staff to perform complex and error prone redeployments of operating systems and applications.
```

Comment:
It seem like you think this needs to be tightened up a bit, which I'm fine with. 

Not to challenge your experience in this space or your experience with Rancher, but I like my phraseology a bit better; except for calling out "security software".

How does it grab you to keep my wording but change "automatically removes all of the previous applications, security software, and policies" to "automatically removes all of the previous applications, and configurations," ?

The idea is, as you've seen, by simply changing the cluster labels, we can quickly repurpose the PoS systems from point-of-purchase to management workstations, to inventory reconciling, to in-store AI video analysis, to ???